mousetrap.js

```diff
@@ -1018,4 +1023,4 @@
             return Mousetrap;
         });
     }
-}) (window, document);
+}) (typeof window !== 'undefined' ? window : null, typeof document !== 'undefined' ? document : null);
```

Comment:
I'm sorry. I've done a bad job with maintaining this. This is an innocent change tho. I'd be happy to merge it